runtime/onert/core/src/ir/pass/PermutationInsertionPass.cc

```diff
@@ -62,27 +62,22 @@ void PermutationInsertionPass::callback(const OperandIndex &index, Operand &obje
     auto insert_set = operand_li->use_factors() - operand_li->def_factors();
     auto def_factor = operand_li->def_factors().getOnlyElement();
 
-    auto compatible_backends = [](auto /* backend1 */, auto /* backend2 */) {
-      // TODO If other issues for Permute elimination are resolved, enable this
-      return false;
-      /*
+    auto compatible_backends = [](auto backend1, auto backend2) {
```

Comment:
IMHO this function is confusing because cpu backend and cf backend are compatible but the reverse is not compatible.

Reply:
This is temporary and it is going to be compatible in both ways. I'll add more description